adding instructions for running app on device in linux

funkyeah opened the pull request.

spencercarli commented on README.md:
```diff
@@ -43,7 +43,35 @@ _Note:_ You have to have the android simulator running before running `react-nat
+
+Build and install on your device, launch the react server, and then launch the app on your phone:
+> react-native android
+> react-native run-android
```
When the markdown is rendered this is showing up on one line. Try using `react-native android && react-native run-android` (using ` for formatting rather than >)

spencercarli commented on README.md:
```diff
@@ -43,7 +43,35 @@ _Note:_ You have to have the android simulator running before running `react-nat
 
-__HELP REQUESTED__: I don't have an Android device so I can't test this out.
+#### For Linux
+##### Enable USB Debugging
```
USB debugging is need for linux or mac, could you move this out of the "For Linux" section and just into the "On a Device" section?

spencercarli commented:
Great start @funkyeah! Just two small comments and we should be ready to merge

funkyeah commented:
addressed your two comments... I actually don't think the `react-native android` step is necessary as long as the project is already android and ios compatible so I removed that (but switched it to us ' formatting anyway)

I also added some details on the setting of the IP address in linux... the genymotion simulator runs in a VM so how to set the IP for that is a bit confusing.

Is running an iOS simulator on OSX the only combination that doesn't need an IP setup step? Maybe abstracting out the IP setup step would be a good idea.

spencercarli commented:
Looks great! I appreciate you help here 😄
